.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
```

Comment:
opt: Maybe call it Tests?

Reply:
I'm also running a linter, so I think its general intent is "stuff that we do on branches to support continuous integration".